extensions/amp-analytics/analytics-vars.md

```diff
@@ -86,6 +92,11 @@ Provides the title of the current document.
 
 Example value: `The New York Times - Breaking News, World News...`
 
+### viewer
+Provides an identifier for the host of the AMP document. Empty string if the document is loaded directly in a browser.
```

Comment:
Might want to reconsider the word "host". A bit overloaded. Will think of alternatives.